client/dposv2/dappchain_dpos_contract.go

```diff
@@ -85,17 +100,17 @@ func (dpos *DAppChainDPOSContract) ChangeFee(identity *client.Identity, candidat
 	return err
 }
 
-func (dpos *DAppChainDPOSContract) ClaimRewards(identity *client.Identity) (*dpostypes.ClaimDistributionResponseV2, error) {
-    req := &dpostypes.ClaimDistributionRequestV2{
-        WithdrawalAddress: identity.LoomAddr.MarshalPB(),
-    }
-    resp := &dpostypes.ClaimDistributionResponseV2{}
-
-    _, err := dpos.contract.Call("ClaimDistribution", req, identity.LoomSigner, resp)
-    if err != nil {
-        return nil, err
-    }
-    return resp, nil
+func (dpos *DAppChainDPOSContract) ClaimRewards(identity *client.Identity, addr loom.Address) (*dpostypes.ClaimDistributionResponseV2, error) {
+	req := &dpostypes.ClaimDistributionRequestV2{
+		WithdrawalAddress: addr.MarshalPB(),
+	}
+	resp := &dpostypes.ClaimDistributionResponseV2{}
+
+	_, err := dpos.contract.Call("ClaimDistribution", req, identity.LoomSigner, &resp)
```

Comment:
So `resp` here is going to be of type `**dpostypes.ClaimDistributionResponseV2`... something is probably going to blow up.

Reply:
Ah typo, gotta remove the `&` from the resp.